src/deno_dir.rs

```diff
@@ -555,18 +555,6 @@ fn test_src_file_to_url_1() {
   );
 }
 
-#[test]
-fn test_src_file_to_url_2() {
-  let (_temp_dir, deno_dir) = test_setup();
-  assert_eq!("hello", deno_dir.src_file_to_url("hello"));
-  assert_eq!("/hello", deno_dir.src_file_to_url("/hello"));
-  let x = deno_dir.deps_https.join("hello/world.txt");
-  assert_eq!(
-    "https://hello/world.txt",
```

Comment:
This is confusing because so much of the test is similar, but this is "https" whereas the one above is "http".